docs/tutorial.md

```diff
@@ -2,165 +2,6 @@
 
 To help illustrate the concepts of NewHELM, we provide a series of functional (if silly) examples in the `demo_plugin` folder. This tutorial walks through those examples.
 
-# Tests
```

Comment:
This diff is actually me moving the previous `tutorial.md` to `tutorial_tests.md`, but it seems `git mv` got ignored.